app/uk/gov/hmrc/tai/util/CeasedEmploymentHelper.scala

```diff
@@ -17,64 +17,15 @@
 package uk.gov.hmrc.tai.util
```

Comment:
Possibly rename to something generic? - now that all the employment specific stuff is gone
I think the CommonVMHelper could also be removed now - and the single utility method moved into somewhere in util. Looks like it's the last trace of the 'VM' stuff ?